Lesson25/Anil/map.js

```diff
@@ -0,0 +1,347 @@
+// Exercise 1:
+// Given an array of strings, convert each string to uppercase using .map().
+const wordsExercise1 = [
+	'apple',
+	'banana',
+	'car',
+	'dog',
+	'elephant',
+	'flower',
+	'guitar',
+	'house',
+	'island',
+	'jungle',
+];
+
+const upperCaseStrings = [...wordsExercise1].map((x)=>x.toUpperCase());
+console.log("uppercaseStrings: ",upperCaseStrings);
+
+
+// Exercise 2:
+// Given an array of numbers, square each number using .map().
+const numbersExercise2 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+const squaredX = array => array.map(x=> x = x**2);
```

Comment:
Pay more attention to naming (avoid using names like x, y and similar). The goal is that the person reading the code can understand what is an expected value of the variable without knowing an actual value.